docs/modules/5_path_planning/time_based_grid_search/time_based_grid_search_main.rst

```diff
@@ -100,9 +100,21 @@ Dynamic Obstacles:
 .. image:: https://raw.githubusercontent.com/AtsushiSakai/PythonRoboticsGifs/refs/heads/master/PathPlanning/TimeBasedPathPlanning/PriorityBasedPlanner/priority_planner.gif
 
 
+Conflict Based Search
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This multi-agent planning algorithm performs searches at two levels. The lower level generates paths for individual agents using a set of constraints provided by the top level search. The top level search generates plans for each agent, and checks for constraints. If any are found, two new nodes are created with one of the two conflicting agents constrained to avoid the (x, y, t) tuple where the conflict occurred.
+
+This process is repeated until the top level search finds a set of path plans with no conflicts. The top level search chooses nodes to expand based on the cost of the paths it has for each agent. This guarantees the solution found is optimal.
+
+This algorithm can also solve some cases that a priority based planner cannot, such as the below example where one robot must move out of the way of another robot's path. Without cooperation across agent paths, this case cannot be solved.
+
+TODO: image waiting on this PR: https://github.com/AtsushiSakai/PythonRoboticsGifs/pull/14
```

Comment:
Will update when this PR merges: https://github.com/AtsushiSakai/PythonRoboticsGifs/pull/14